cmd/influx/ping.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/influxdata/influxdb/kit/check"
+	"github.com/spf13/cobra"
+	"io/ioutil"
+	"net/http"
+	"time"
+)
+
+var pingCmd = &cobra.Command{
+	Use:   "ping",
+	Short: "Check the InfluxDB /health endpoint",
+	Long:  `Checks the health of a running InfluxDB instance by querying /health. Does not require valid token.`,
+	RunE:  pingF,
+}
+
+func pingF(cmd *cobra.Command, args []string) error {
+	if flags.local {
+		return fmt.Errorf("local flag not supported for ping command")
+	}
+
+	c := http.Client{
+		Timeout: 5 * time.Second,
+	}
+	url := flags.host + "/health"
+	resp, err := c.Get(url)
+	if err != nil {
+		return err
+	}
+	if resp.StatusCode/100 != 2 {
+		return fmt.Errorf("got %d from '%s'", resp.StatusCode, url)
+	}
+
+	b, err := ioutil.ReadAll(resp.Body)
```

Comment:
Prefer `json.NewDecoder(resp.Body).Decode(&healthResponse)` so that you don't have to read the entire response body up-front (i.e. don't trust the network).